### ISSUE TITLE
Create interface between the Pi server and UI

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#17]
## User story
1. As a SD
2. I want to connect the RasPi server and UI
3. So that I can transfer data between these components

## Acceptance criteria
* Functional interface between these components

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by UmangBR** [comment]
Might be possible with Flask and Ajax - A easy and Robust framework to implement

**Comment by cat24max** [comment]
Duplicate #19

**Comment by UmangBR** [comment]
> Duplicate #19

Please confirm the following: 
1) RasPi server = FMS backend and UI = FMS front end.
2)#17 is the first version of interface between above mentioned components and needs to be further expanded/modified. This modifaction will depend on how FMS and robot communicate.

**Comment by UmangBR** [comment]
To be continued in #19
